src/contracts/exchange/NiftyswapExchange.sol

```diff
@@ -37,7 +37,7 @@ contract NiftyswapExchange is ReentrancyGuard, ERC1155MintBurn, INiftyswapExchan
   bool internal currencyPoolBanned;               // Whether the currency token ID can have a pool or not
   address internal factory;                       // address for the factory that created this contract
   uint256 internal currencyID;                    // ID of currency token in ERC-1155 currency contract
-  uint256 internal constant FEE_MULTIPLIER = 995; // Multiplier that calculates the fee (0.5%)
+  uint256 internal constant FEE_MULTIPLIER = 995; // Multiplier that calculates the fee (1.0%)
```

Comment:
this is hardcoded, but should it be a variable adjustable only on construction..? with our factory, we have just a single instance for this pair, which keeps things simple and nice, but what if we ever want to adjust these numbers unilaterally across contracts..? how could that work?

Reply:
>  what if we ever want to adjust these numbers unilaterally across contracts..? how could that work?

We would need to deploy a new factory with a new fee and then people would need  deploy the exchange and migrate their liquidity, like with Uniswap. It's important the LP fee can't change under people's feet.